src/PolymathAPI.ts

```diff
@@ -206,6 +211,14 @@ export class PolymathAPI {
     });
   };
 
+  /**
+   * Get a wrapped token from an address
+   * @return TokenWrapper ERC20
+   */
+  public getERC20TokenWrapper = async (params: GetERC20WrapperParams): Promise<ERC20> => {
+    return this.tokenFactory.getERC20TokenInstanceFromAddress(params.address);
```

Comment:
Shouldnt this be:
```
 public getERC20TokenWrapper = async (params: GetERC20WrapperParams): Promise<ERC20TokenWrapper> => {
```

Reply:
Why? The class is imported as `ERC20` intentionally to match what is being exported by the index

Reply:
Oh I see